lib/sufia/engine.rb

```diff
@@ -1,4 +1,12 @@
 module Sufia
+  class Statefile
+    def self.default
+      return '/tmp/minter-state' if Rails.env.development? || Rails.env.test?
+      raise NotImplementedError, "can't use the default statefile for production" unless Dir.exist?('/var/sufia')
```

Comment:
@mjgiarlo see @jcoyne above. He's correct. Since the engine code gets executed first, Sufia.config doesn't exist yet, so we don't really know what the user has specified for their statefile. The only solution I can see right now is perhaps raising an error if `/tmp/minter-state` exists while running in production, or just returning `nil` as the default in production which would force the user to set it to something. Or both?